src/tasks.js

```diff
@@ -0,0 +1,98 @@
+// Add task button
+const addTask = document.querySelector('#add_task');
+const inputDesc = document.querySelector('#task_description');
+const tasksArr = JSON.parse(localStorage.getItem('tasksArr')) || [];
+class Tasks {
+  constructor(description, completed, index) {
+    this.index = index;
```

Comment:
Please make sure the property `index` is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5).